sagemaker/launch/common_parser.py

```diff
@@ -49,7 +49,7 @@ def get_common_parser() -> argparse.ArgumentParser:
     common_args.add_argument("--instance-type", type=str,
         help="instance type for the SageMaker job")
     common_args.add_argument("--instance-count", type=int,
-        default=2,
+        default=1,
```

Comment:
In any case we will use the common parser default? I assume we will only call the script via launch_TASK.py.

Reply:
All `launch_*.py` tasks use the common parser, so they use the default here. Since now we have at least one job that requires a single instance I thought we should change the default.

Ideally I'd prefer this to be a required argument but we can discuss that elsewhere.